app/views/spree/orders/_pay_easy.html.erb

```diff
@@ -0,0 +1,8 @@
+<div>
+  Thank you for your order. Instructions have been sent to your e-mail address.<br>
+  To complete a payment at <strong><%= link_to "purchase page", source.pay_url %></strong>.<br>
+  <br>
+  <%= link_to source.instructions_url do %>
+    How to make a payment.
```

Comment:
Also, the link points to this page: http://cvs.komoju.com/payeasy.html

But this might be confusing since we only offer payment by internet banking, not by ATM...